### ISSUE TITLE
API search using applicationIds does not return all the vulnerabilities

### ISSUE THREAD

**Issue by continuumsecurity** [denimgroup/threadfix#1471]
When searching using the CLI with command:

java -jar tfcli.jar --search applicationIds=1

The returned results do not include many vulnerabilities listed in the GUI, for example SQL injection vulnerabilities reported in the GUI are not reported by the API results.
But re-running the search specifically searching for SQL injection vulnerabilities.

java -jar tfcli.jar --search genericVulnerabilityIds=89 applicationIds=1

Results in the missing vulnerabilities being displayed.

In short, java -jar tfcli.jar --search applicationIds=1 does not report all vulnerabilities for that app.

**Comment by continuumsecurity** [comment]
This seems to be a default limit based on the number of vulnerabilities.  Using the following results in all the vulnerabilities returned:

java -jar tfcli.jar --search applicationIds=1 numberVulnerabilities=5000
